.github/workflows/test-pr.yml

```diff
@@ -8,10 +8,10 @@ jobs:
   test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
         with:
-          node-version: '14'
+          node-version: '18'
       - run: npm install -g npm@7 --registry=https://registry.npmjs.org
```

Comment:
Installing npm should not be needed anymore, and this code actually pinned npm to an obsolete version of npm.

```suggestion
```

(Same comment applies to the other job, but I cannot "suggest" the change through the GitHub UI because the line is too far from the touched update ;))